shopfloor_packing_info/views/stock_picking_views.xml

```diff
@@ -10,23 +10,15 @@
                 position="before"
             >
                 <group name="shopfloor">
-                    <field name="shopfloor_display_packing_info" invisible="1" />
```

Comment:
If I understand correctly, `shopfloor_display_packing_info` will be dropped entirely, do we really want this?

Reply:
Yes the purpose is to decrease shopfloor complexity.
This gets replaced by the picking note that dan be populated with stock_picking_partner_note